geometry_msgs/Vector3 output_vector